src/sotoki/entrypoint.py

```diff
@@ -113,7 +113,7 @@ def main():
         "-p",
         "--publisher",
         help="Custom publisher name (ZIM metadata). “OpenZIM” otherwise",
-        default="Kiwix",
+        default="openZIM",
```

Comment:
I know it was here before you came, but please remove this default value, we should not have two default values in two different locations, one is enough

Reply:
@benoit74 I have made the changes you requested. Can you please review it?

Reply:
Thank you for the ping, looks like the notification slipped through my attention. I will have a look at it right now.